ziti-springboot-client/src/main/java/org/openziti/springboot/client/web/config/ZitiHttpClientConfiguration.java

```diff
@@ -171,8 +168,9 @@ public ConnectionKeepAliveStrategy connectionKeepAliveStrategy(
     };
   }
 
-  @Bean("zitiHttpClient")
-  public CloseableHttpClient httpClient(
+  @ConditionalOnProperty(value = "spring.ziti.client.http-client.enabled", havingValue = "true", matchIfMissing = true)
+  @Bean
+  public CloseableHttpClient zitiHttpClient(
```

Comment:
personally, i feel like developers are much easier to create a backward compatibility problem by relying on the method name. one refactor and suddenly it's no longer named "zitiHttpClient". I prefer the more cautious approach for that reason

Reply:
You mean by adding `@Bean("zitiHttpClient")` instead? it's the same outcome.

Reply:
yes it's the same outcome but if a dev refactors zitiHttpClient to anything else, the bean name won't change. I'd prefer to see the bean name in the bean for that reason